pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -669,9 +670,16 @@ def _get_org_id(request):
 
     def count_download(filename):
         ns, name, _ = filename.split("-", maxsplit=2)
-        collection, _ = CollectionDownloadCount.objects.get_or_create(namespace=ns, name=name)
-        collection.download_count = F("download_count") + 1
-        collection.save()
+        try:
+            collection, _ = CollectionDownloadCount.objects.get_or_create(namespace=ns, name=name)
+            collection.download_count = F("download_count") + 1
+            collection.save()
```

Comment:
```suggestion
            collection, created = CollectionDownloadCount.objects.get_or_create(namespace=ns, name=name, defaults={"download_count": 1})
            if not created:
                collection.download_count = F("download_count") + 1
                collection.save()
```

Reply:
Although this suggestion does reduce the overall db calls, the model's default value is zero and this new code causes tests to fail because the first download doesn't increment the counter.

```
>           assert (download_count + 1) == get_current_download_count(                                                                      
                api_client=galaxy_v3_collections_api_client,          
                path=install_scenario_distribution.base_path,         
                namespace=collection_namespace,                       
                name=collection_name,                                 
            )                                                         
E           AssertionError: assert (0 + 1) == 0 
```

Reply:
But we could change that default to 1. I think this was implied. Sorry.

Reply:
We do have it on in some scenarios:
https://github.com/pulp/pulp_ansible/blob/main/template_config.yml#L60

And on close inspection, the suggestion overwrites the default inline (`defaults={"download_count": 1}`) with the `get_or_create` that's why i think it should pass the tests.